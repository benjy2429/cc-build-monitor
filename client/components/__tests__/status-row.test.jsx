import React from 'react';
import { shallow } from 'enzyme';
import StatusRow from '../status-row';

const defaultProps = {
  projects: [
    {
      buildStatus: 'Failure',
      activity: 'Sleeping',
      name: 'org/project-1',
    },
    {
      buildStatus: 'Failure',
      activity: 'Sleeping',
      name: 'org/project-2',
    },
  ],
  status: 'Failing',
};

describe('Status row', () => {
  it('renders correctly', () => {
    const component = shallow(
      <StatusRow {...defaultProps} />,
    );

    expect(component).toMatchSnapshot();
  });
});
