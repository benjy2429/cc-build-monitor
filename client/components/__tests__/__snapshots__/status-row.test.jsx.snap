// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status row renders correctly 1`] = `
<div
  className="status-row"
>
  <h2
    className="status-heading"
  >
    2
     
    Failing
  </h2>
  <div
    className="status-projects"
  >
    <Project
      project={
        Object {
          "activity": "Sleeping",
          "buildStatus": "Failure",
          "name": "org/project-1",
        }
      }
    />
    <Project
      project={
        Object {
          "activity": "Sleeping",
          "buildStatus": "Failure",
          "name": "org/project-2",
        }
      }
    />
  </div>
</div>
`;
